src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,187 @@
+      allSizes.push(secondSize);
+      const thirdSize = e.target["thirdSize"].value;
+      allSizes.push(thirdSize);
+      const fourthSize = e.target["fourthSize"].value;
+      allSizes.push(fourthSize);
+    }
+
+    const productInfo = {
+      brand: getProductBrand,
+      title: getProductTitle,
+      categories: getProductCategory,
+      description: getProductDescription,
+      images: allImages,
+      hasSize: hasSizes,
+      sizes: allSizes,
+      price: getProductPrice,
+    };
+
+    await db.collection("products").add(productInfo);
+  };
+
+  return (
+    <>
+      <Form onSubmit={(e) => addNewProductToDatabase(e)}>
+        <Container>
+          <Row>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Brand"
+                onChange={(e) => setGetProductBrand(e.target.value)}
+              />
+            </Col>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Title"
+                onChange={(e) => setGetProductTitle(e.target.value)}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col>
+              <Form.Control
+                type="text"
+                placeholder="Description"
+                onChange={(e) => setGetProductDescription(e.target.value)}
+              />
+            </Col>
+          </Row>
+          <Row>
+            <Col xs={12} md={6} lg={5}>
+              <Form.Group controlId="formGridState">
+                <Form.Control
+                  as="select"
+                  defaultValue="Shoes"
+                  onChange={(e) => setGetProductCategory(e.target.value)}
+                >
+                  <option>Shoes</option>
+                  <option>Health care</option>
+                </Form.Control>
+              </Form.Group>
+            </Col>
+            <Col xs={12} md={6} lg={3} className="radioBtn">
+              <Form.Check
+                type="radio"
+                label="Size"
+                name="formHorizontalRadios"
+                id="formHorizontalRadios2"
+                onChange={() => setHasSizes(true)}
+              />
+              <Form.Check
+                type="radio"
+                label="No size"
+                name="formHorizontalRadios"
+                id="formHorizontalRadios1"
+                onChange={() => setHasSizes(false)}
+              />
+            </Col>
+            <Col xs={12} md={6} lg={4}>
+              <Form.Control
+                type="number"
+                placeholder="Price"
+                onChange={(e) => setGetProductPrice(parseInt(e.target.value))}
+              />
+            </Col>
+          </Row>
+          {hasSizes && (
+            <Row>
+              <Col xs={12} md={6} lg={3}>
+                <Form.Control as="select" defaultValue="sizes" id="firstSize">
+                  <option>45</option>
```

Comment:
Can you create a variable that is consisting of `<option>` elements to avoid using same code blocks again.